frb/figures/finder.py

```diff
@@ -84,6 +86,9 @@ def generate(image, wcs, title, flip_ra=False, flip_dec=False,
         secondary_coord (astropy.coordinates.SkyCoord, optional):
           If provided, place a mark in cyan at this coordinate
           Assume it is an offset star (i.e. calculate offsets)
```

Comment:
"It assumes it is an offset star (i.e. calculate offsets) unless `secondary_offset` is set to `False`"

Reply:
revised